### ISSUE TITLE
fails to build on clang with `undefined reference to 'rs_appendflush'`

### ISSUE THREAD

**Issue by sourcefrog** [librsync/librsync#41]
On Debian Jessy with 

> Debian clang version 3.5.0-10 (tags/RELEASE_350/final) (based on LLVM 3.5.0)

```
libtool: link: clang -Wall -Wshadow -Wundef -Wwrite-strings -Wmissing-prototypes -Wmissing-declarations -Wstrict-prototypes -Wpointer-arith -Wcast-qual -Wcast-align -g -O2 -o rdiff rdiff.o isprefix.o  ./.libs/librsync.a /usr/lib/x86_64-linux-gnu/libpopt.so -lbz2 -lz
./.libs/librsync.a(delta.o): In function `rs_appendmatch':
/home/mbp/src/librsync/delta.c:294: undefined reference to `rs_appendflush'
./.libs/librsync.a(delta.o): In function `rs_appendmiss':
/home/mbp/src/librsync/delta.c:322: undefined reference to `rs_appendflush'
./.libs/librsync.a(delta.o): In function `rs_appendmatch':
/home/mbp/src/librsync/delta.c:294: undefined reference to `rs_appendflush'
./.libs/librsync.a(delta.o): In function `rs_appendmiss':
/home/mbp/src/librsync/delta.c:322: undefined reference to `rs_appendflush'
./.libs/librsync.a(delta.o): In function `rs_delta_s_flush':
/home/mbp/src/librsync/delta.c:221: undefined reference to `rs_appendflush'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Makefile:594: recipe for target 'rdiff' failed
```

It might be unhappy with something about the `inline` declaration.

**Comment by sourcefrog** [comment]
This might also be a bug in the automake setup and fixed by switching away from that in #39.

**Comment by Lessica** [comment]
export CFLAGS="${CFLAGS} -std=gnu89"

**Comment by Salamek** [comment]
#27 builds ok with clang https://travis-ci.org/Salamek/librsync/jobs/69100039 see https://github.com/Salamek/librsync/commit/1175b1cd102ee15512185c7983868e5746959da0

**Comment by Salamek** [comment]
@sourcefrog should be fixed by 94ce627c7832dae8dd4231491cfa5e035fcb2771 and travis clang build went [OK](https://travis-ci.org/librsync/librsync/jobs/91262501) so i think you can close this...

**Comment by sourcefrog** [comment]
yep, thanks
